eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -48,16 +48,31 @@ jobs:
 
       - template: ../steps/common.yml
 
-      - script: |
-          node common/scripts/install-run-rush.js install
-        displayName: "Install dependencies"
-
       - pwsh: |
           $folder = "${{parameters.ServiceDirectory}}"
           if ($folder -eq "*") { $folder = "" }
           echo "##vso[task.setvariable variable=folder]$folder"
         displayName: "Set folder variable for readme links"
 
+      - pwsh: |
+          $setDailyDevBuild = "false"
+          if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
```

Comment:
An override would make it so we could validate this functionality by manually setting the `DevBuildRun` variable when queuing. 

```
if ($(DevBuildRun) -eq '') {
  exit
}

$setDailyDevBuild = "false" 
if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
...

```

Or it might just make sense to set a condition on the script itself if this is the only variable it sets. 
```
condition: eq(variables['DevBuildRun'], '')
```

Reply:
I agree we want some sort of override but I was going to try and make it consistent across the different pipelines which is why I didn't suggest one yet for this one.  Perhaps we can use the name `SetDevVersion` as the property across all the languages. @KarishmaGhiya  please go ahead and update based on @danieljurek's suggestion but change the variable to `SetDevVersion`.

Reply:
@danieljurek Can we not compare this on true instead of "" ?

Reply:
Thanks @KarishmaGhiya for pointing out the contradictory logic... The `pwsh` syntax SHOULD have used `-ne` in the first if statement...

```
if ($(DevBuildRun) -ne '') {
  exit
}

$setDailyDevBuild = "false" 
if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
...
```

Reply:
done